modules/yaml-config/main.tf

```diff
@@ -5,7 +5,14 @@ locals {
   ] : []
 
   # Terraform maps from local YAML configuration templates
-  local_map_configs = flatten(
+  local_map_configs = flatten([
```

Comment:
`local_map_configs` looks like not used anywhere in the code

Reply:
Thanks, repurposed this local with the `_decoded` because of output types and ended up not using this after testing